cp2/ControlPanel/ControlPanel/utils.py

```diff
@@ -10,30 +10,25 @@
 logger = logging.getLogger(__name__)
 
 
-def api_post(resource, method, params=None, files=None, json=None):
+def api_post(resource, method, **kwargs):
```

Comment:
Doesn't this change break a lot of `api_post` calls?
Also I think there are still calls to `api_post` that do not provide the method argument (see `git grep api_post -- cp2`).